src/resolve/resolved-resolv-conf.c

```diff
@@ -179,7 +179,7 @@ static void write_resolv_conf_server(DnsServer *s, FILE *f, unsigned *count) {
         assert(count);
 
         if (!dns_server_string(s)) {
-                log_warning("Our of memory, or invalid DNS address. Ignoring server.");
+                log_warning("Out of memory, or invalid DNS address. Ignoring server.");
```

Comment:
We shouldn't be lazy like that, `dns_server_string` should return a proper error. But that's a bigger fix, your patch is valid too.